src/cloud/pricing/price-plan.md

```diff
@@ -0,0 +1,27 @@
+# Cloud Pricing
+
+## Free Tier
+
+In PGVecto.rs Cloud, we provide a free tier that allows you learn and explore fully-managed PGVecto.rs services in a cloud environment. Every user can use the free tier to create **only one** cluster and store vector data in it. There are some limitations in the free tier:
+- Shared resources
+- Limited storage for WAL (5Gi)
+- Only support AWS T3 XLarge instance type
+- Capacity of 100k vectors with 1024 dimensions 
```

Comment:
Does it support 25k with 4096 dim? Users may think that the 1024 dim is a hard limit.